src/module/PrcaOrchestrator.ts

```diff
@@ -111,4 +116,15 @@ export class PrcaOrchestrator {
         return result;
     }
 
+    /* Static methods */
+
+    public static CreatePrcaOrchestrator(logger: ILogger, collectionUrl: string, token: string, repositoryId: string, pullRequestId: number): PrcaOrchestrator {
```

Comment:
It's an antipattern to have both static builders and constructors. Since we cannot have multiple ctors, at least have both those methods be static builders